src/anemoi/models/layers/attention.py

```diff
@@ -11,19 +11,11 @@
 from typing import Optional
 
 import einops
+from flash_attn import flash_attn_func
```

Comment:
When is the import triggered? Only when building models with attention?

Reply:
good point, I will put into the init instead